src/cmd/cmdfuel.rb.in

```diff
@@ -79,7 +80,10 @@ SUBCOMMANDS = {
   "  -u [--url] arg           Full resource URL, such as:                  \n"\
   "                           https://fuel.ignitionrobotics.org/1.0/OpenRobotics/models/Ambulance\n"\
   "  --header arg             Set an HTTP header, such as                  \n"\
-  "                           --header 'authorization: Bearer JWT'.        \n" +
+  "                           --header 'authorization: Bearer JWT'.        \n"\
+  "  -j [--jobs] arg          Number of parallel downloads (default: 1, max: #{MAX_PARALLEL_JOBS}). \n"\
+  "  -t [--type] arg          Select which resource type (i.e. model, world) \n" \
+  "                           to download from collection. Ignored if not downloading collection. \n" +
```

Comment:
Maybe it's worth noting that all resources in the collection will be downloaded if the type is not specified.

Maybe something like "Limit what resource type to download from a collection"?

Reply:
Good idea. https://github.com/ignitionrobotics/ign-fuel-tools/pull/98/commits/5a90f5a36adee164ccdb1fcb2a3622f561cb61b5